test/helpers/cilium.go

```diff
@@ -343,14 +359,74 @@ func (c *Cilium) ServiceAdd(id int, frontend string, backends []string, rev int)
 	return c.Exec(cmd)
 }
 
+// ServiceDel deletes the service with the specified ID
+func (c *Cilium) ServiceDel(id int) *CmdRes {
+	return c.Exec(fmt.Sprintf("service delete '%d'", id))
+}
+
+// ServiceIsSync checks that the service is sync in the bpf lb list
+func (c *Cilium) ServiceIsSync(id int) (bool, error) {
+	var svc *models.Service
+	svcRes := c.ServiceGet(id)
+	if !svcRes.WasSuccessful() {
+		return false, fmt.Errorf("cannot get service id %d: %s", id, svcRes.CombineOutput())
+	}
+	err := svcRes.UnMarshal(&svc)
+	if err != nil {
+		return false, err
+	}
+
+	bpfLB, err := c.BpfLBList()
+	if err != nil {
+		return false, err
+	}
+
+	frontendAddr := net.JoinHostPort(
+		svc.FrontendAddress.IP,
+		fmt.Sprintf("%d", svc.FrontendAddress.Port))
+	lb, ok := bpfLB[frontendAddr]
+	if ok == false {
+		return false, fmt.Errorf("service is not sync with BPF lb")
```

Comment:
Specify how it is not in sync (frontend address from the service with the specified ID does not have a corresponding frontend address in the bpf maps, etc.) along with which service being analyzed resulted in this error. Otherwise, in logs, we won't be able to determine which service resulted in this error unless the caller of this function does so.